PORTING_TO_AWSLC.md

```diff
@@ -1,7 +1,5 @@
 # Porting from OpenSSL/BoringSSL to AWS-LC
 
-AWS-LC іs based on code from the Google BoringSSL project and the OpenSSL project.
```

Comment:
Why is this considered duplicate?

Reply:
I copied from README.md then thought the copy was duplicate.

Reply:
I get it now. I didn't know this sentence was already in README.md. Makes sense.